app/cache-freshener.sh

```diff
@@ -22,7 +22,7 @@ curl -s ${LINKED_CONTAINER_NAME}:${PORT} > /json/index.html
 echo "Wrote base HTML file"
 
 # Add the service-list endpoint to the list of things to update
-ENDPOINTS="${ENDPOINTS} service-list"
+ENDPOINTS="${ENDPOINTS} service-list all"
```

Comment:
Hmmm...I don't want it running this on every cache update - that's why I removed it...

Is there a way to have it accessible, but not update the cache for it everytime?

Reply:
Hmm, maybe some kind of multithreading/multiprocessing that loops separately for two sets of endpoints with different timers.

Reply:
I was thinking more along the lines of not maintaining a cache for this endpoint, and only fetching it on-the-fly when requested